Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
-    <string name="media_settings_do_you_want_to_copy_files">Do you want to copy %1$d files from old directory to the new one?"</string>
-    <string name="media_settings_move_saved_files_to_new_dir">Move old saved files to the new directory?</string>
-    <string name="media_settings_move_saved_files_to_new_dir_description">Move files from \n\n\"%1$s\"\n\n directory to \n\n\"%2$s\"\n\ndirectory? This operation may take quite some time. Once started this operation must not be canceled. \nThis operation will move ALL the files, so if you had some files not related to this app you probably should do that manually.</string>
-    <string name="media_settings_new_saved_files_base_dir_not_registered">New saved files base directory is probably not registered</string>
+    <string name="media_settings_do_you_want_to_copy_files">Copy %1$d files from old location to the new one?"</string>
+    <string name="media_settings_move_saved_files_to_new_dir">Move old saved files to the new location?</string>
+    <string name="media_settings_move_saved_files_to_new_dir_description">Move files from \n\n\"%1$s\"\n\n to \n\n\"%2$s?\"\n\n This may take quite some time and must not be canceled once started. \nMoves ALL the files, so if you have some files unrelated to this app you should do so manually.</string>
+    <string name="media_settings_new_saved_files_base_dir_not_registered">The base folder for saved files is probably not registered</string>
     <string name="media_settings_no_files_to_copy">No files to copy</string>
-    <string name="media_settings_copying_file">Copying file %1$d out of %2$d</string>
-    <string name="media_settings_cannot_continue_write_permission">Cannot continue without the WRITE_EXTERNAL_STORAGE permission</string>
-    <string name="media_settings_base_directory_is_already_registered">Cannot use this directory because it is already in use</string>
+    <string name="media_settings_copying_file">Copying file %1$d of %2$d…</string>
+    <string name="media_settings_cannot_continue_write_permission">Grant write permission to external storage first.</string>
+    <string name="media_settings_base_directory_is_already_registered">Cannot use this a folder already in use</string>
 
     <string name="add_dubs_title">Add dubs to post cells</string>
     <string name="add_dubs_description">Appends your GET to your post number, for FUN!</string>
     <string name="files_base_dir_does_not_exist"><![CDATA[Base directory for files does not exist! Set it manually in the settings (Media -> Save location)]]></string>
-    <string name="image_saver_could_not_figure_out_save_location">Couldn\'t figure out save location</string>
-    <string name="settings_captcha_setup">Captcha Settings</string>
-    <string name="settings_captcha_setup_description">Captcha settings are per-site configurable. Tap for site setup.</string>
+    <string name="image_saver_could_not_figure_out_save_location">Could not figure out where to save this</string>
```

Comment:
Don't use a generic ``this``.

Reply:
Neither is very good. Will revise.
Is the user prompted to select a folder afterwards?